ios/Capacitor/Capacitor/CAPUNUserNotificationCenterDelegate.swift

```diff
@@ -42,14 +42,24 @@ public class CAPUNUserNotificationCenterDelegate : NSObject, UNUserNotificationC
     let request = notification.request
     var plugin: CAPPlugin
     var action = "localNotificationReceived"
-    print(notification)
+    var presentation_options: UNNotificationPresentationOptions = [];
+
     var notificationData = makeNotificationRequestJSObject(request)
     if (request.trigger?.isKind(of: UNPushNotificationTrigger.self))! {
       plugin = (self.bridge?.getOrLoadPlugin(pluginName: "PushNotifications"))!
       action = "pushNotificationReceived"
       notificationData = makePushNotificationRequestJSObject(request)
+      presentation_options = [
+        UNNotificationPresentationOptions.badge,
```

Comment:
No need to add the `UNNotificationPresentationOptions` , this can be one line `presentationOptions = [.badge, .sound]`

Same for the other with the .alert